docs/topics/modules/using_stack_reports.adoc

```diff
@@ -1,27 +1,46 @@
 [id="using_stack_reports"]
-= Using stack reports
+= Using stack report
 
-{osio} includes a Stack Report feature that provides information about your stack components and recommends improvements for your application. The Stack Report is available after the first pipeline build completes.
+The stack report provides you with data about your stack and its components and provides recommendations based on its analysis of your stack. It enables you to assess the credibility of your components and enhance your stack.
 
-. Return to your space dashboard in the {osio} browser tab.
-. The `Stack Report Recommendations` panel in the dashboard shows a summary of the Stack Report.
-+
-image::stack_reports.png[Stack Reports]
-+
-. Click *Show complete stack report* at the bottom of this panel to view the complete report.
-+
-image::full_stack_report.png[Full Stack Report]
-+
-The `Additional components recommended by {osio}` section at the bottom has one recommendation to improve the application. It includes a link to create a new Work Item to track making the changes for this recommendation:
+== Components of stack report
+The {osio} analytics report comprises three components:
+
+=== Stack summary
+The stack summary provides a synopsis of the analysis report.
+It highlights the security issues, <<glossary,usage outliers>>, and unknown or conflicting Licenses in your stack.
```

Comment:
You can't link to the Glossary like this. Either remove the xref, or use `link:` like I showed you in some previous comment.